modules/openshift-developer-cli-configuring-debugging-parameters.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/debugging-applications-in-odo.adoc
+
+[id="configuring-debugging-parameters_{context}"]
+
+= Configuring debugging parameters
+
+.Procedure
+
+* To set a remote port on which the debugging agent should run, use:
++
+----
+$ odo config set DebugPort 9292
+----
+
```

Comment:
Should add a + on this line so that the following note is indented properly with the above bullet.